architecture/users/readme.md

```diff
@@ -56,12 +50,12 @@
             "name" : "..."
         },
         ...
-    ]    
+    ]
     ```
 
-### USR-F2 Я как пользователь хочу иметь возможность получить пользователей по их uuid
+### USR-F2 I, as a user, want to be able to get list of students on their uuid
```

Comment:
~I, as a user, want to be able~ **As a user I want** to get list of students ~on~ **by** their uuid